### ISSUE TITLE
[INCOMPATIBILITY] Wall-Jump Mod. After a grappling hook swing and release, you cannot wall jump or wall cling.

### ISSUE THREAD

**Issue by Breadzticks** [yyon/grapplemod#102]
**Describe the bug**
 After a grappling hook swing and release, you cannot wall jump or wall cling. Upon releasing and landing on vines, you cannot climb, and upon reaching the ground you take fall damage (seemingly equivalent to if you were free falling that entire time). 
 

**Video / Screenshots**

**Version**
grappling_hook_mod-1.18.2-v13.jar
minecraft 1.18.2
walljump-forge-1.18.1-1.3.7.jar

**Output Log**
N/A - It is a gameplay interaction or player-state bug, not a bug that crashes the game.

**To Reproduce**
Install both Wall Jump and Grappling hook mod. After swinging, release and then touch a wall. Press the wall cling button (Shift for me).

Additional related bug - Swing and release, onto a wall of Vines, or a ladder. You'll be unable to climb and upon reaching the ground, you take enormous fall damage. 

**Additional context**
My hypothesis is that after releasing the grappling hook, it puts your character in a state that makes him unable to do certain things. Perhaps you are forcing a free falling state until the player lands on the ground?

**Comment by Breadzticks** [comment]
Update -- I've tried many other interactions, from other mods, to even vanilla interactions that require you to be in an airborne state to work, so it may not be an incompatibility between mods and leads me further to believe its a player state issue.

**Comment by ghost** [comment]
Yeah, there's an issue where its not treated like an actual airborne state. For instance, Elytra just will not open at all if you try to open the wings. Its a very frustrating problem to have. You can fix it by setting the `dont_override_movement_in_air` setting. That said its... really not a problem the mod should have imo.
